src/libraries/Native/AnyOS/entrypoints.h

```diff
@@ -0,0 +1,34 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+#pragma once
+
+#include <stdint.h>
+#include <string.h>
+
+#ifndef lengthof
+#define lengthof(rg) (sizeof(rg)/sizeof(rg[0]))
```

Comment:
Nit: is there a reason to use a macro here instead of an inline function? In general, I prefer to avoid macros when not building new syntax.

Reply:
This is not a new macro. It is a copy of existing one. And we have several more versions of this under different names as well. It was discussed in this PR. 
It may be a good idea to just introduce one helper in a place that everyone has access to and use that throughout the codebase. Not doing such change here though.

Reply:
Oh I see, it's implemented as a macro all throughout CoreCLR :(

I still think using macros in place of inline functions is not a very good pattern, but if it's an established one within CoreCLR I guess that makes sense. Not a huge deal either way.

Reply:
@CoffeeFlux What would be the C99 implementation of `lengthof`? I have only ever seen this implemented as macro in C.